core/templates/services/context.service.spec.ts

```diff
@@ -96,9 +114,22 @@ describe('Context service', () => {
       expect(ecs.getEntityType()).toBe('exploration');
     });
 
+    it('should affirm the page context to be exploration player or editor',
+      ()=> {
+        expect(ecs.isInExplorationContext()).toBe(true);
+      });
+
     it('should affirm that the page forbids editing of RTE components', () => {
       expect(ecs.canAddOrEditComponents()).toBe(false);
     });
+
+    it('should correctly affirm that page contains reference to skills',
+      () => {
+        expect(ecs.getPageContext()).toBe('learner');
+        expect(ecs.canEntityReferToSkills()).toBe(false);
+        ecs.setExplorationIsLinkedToStory();
+        expect(ecs.canEntityReferToSkills()).toBe(false);
+      });
```

Comment:
This whole block is duplicated? Please avoid copy-pasting tests.. running unwanted tests actually slows down our CI / dev flow.

Reply:
Removed the block. Done!